README.md

```diff
@@ -222,3 +222,63 @@ logger.error('error message', { error: new Error('error') });
 ```
 
 </details>
+
+## Monday API Client
+
+<details>
+<summary>Querying the monday.com GraphQL API seamlessly on behalf of the connected user, or using a provided API token.</summary>
+
+#### initialize
+
+```typescript
+import { MondayApiClient } from '@mondaycom/apps-sdk';
+
+const mondayClient = new MondayApiClient();
+mondayClient.setToken(token);
+mondayClient.setApiVersion('2023-10');
+```
+
+```typescript
+import { MondayApiClient } from '@mondaycom/apps-sdk';
+
+const mondayClient = new MondayApiClient(token);
+mondayClient.setApiVersion('2023-10');
+```
+
+#### API
+
+```typescript
+const query = `query {
+        items(ids: [${itemId}]) {
+          column_values(ids: ["${columnId}"]) {
+            text
+          }
+        }
+      }`;
+
+logger.debug(`getApiToGetColumnValue->query:
+    ${query}
+    `);
+const response = await mondayClient.api(query);
```

Comment:
Why don't you use variables here like in the mutation? this would look like we don't recommend using variables in queries (like in lines `265~281`)